app/models/vm.rb

```diff
@@ -1,6 +1,7 @@
 class Vm < VmOrTemplate
   default_scope { where(:template => false) }
-
+  has_one :container_deployment, :through => :deployments_node_deployment
+  has_one :container_node_deployment
```

Comment:
I don't see it as a problem, its just a link, and we aren't doing any changes on those tables.